src/admin-views/aggregator/tabs/import-form.php

```diff
@@ -1,5 +1,6 @@
 <?php
-$has_license_key = Tribe__Events__Aggregator::is_service_active();
+$has_license_key = tribe( 'events-aggregator.main' )->is_service_active();
```

Comment:
Yep, or what @bordoni said (but that doesn't need to happen as part of this change)